apps/admin-ui/src/spa/ReservationUnit/edit/index.tsx

```diff
@@ -153,6 +153,19 @@ const SubAccordion = styled(Accordion)`
   }
 `;
 
+const BufferWrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-column: 2 / 13;
```

Comment:
isn't `2 / 13` === `2 / -1`? meaning it skips the first column, but fills everything else?
Just a preference, but -1 is better for the last column (and it's currently used in the code).

Reply:
Good catch! Yeah, skips first column, and fills out the rest of the space.